docpages/example_programs/misc.md

```diff
@@ -8,3 +8,4 @@ This section lists examples that do not fit neatly into any of the categories ab
 * \subpage collecting-reactions "Collecting Reactions"
 * \subpage cpp-eval-command-discord "Making an eval command in C++"
 * \subpage checking-member-permissions "Checking permissions"
+* \subpage user-only-messages.md "Hidden replies (Only you can see this replies)"
```

Comment:
the replies are called ephemeral messages in discord.
perhaps the title should be:
`Ephemeral Messages ("Only you can see this" replies)`

Reply:
I would move it to interaction examples as it's a slashcommand example 😉